src/game/boe.actions.cpp

```diff
@@ -2454,18 +2494,29 @@ void init_debug_actions() {
 	add_debug_action({'<'}, "Make one day pass", debug_increase_age);
 	add_debug_action({'>'}, "Reset towns (excludes the one you're in, if any)", debug_towns_forget);
 	add_debug_action({'!'}, "Toggle Special Node Step-through Mode", debug_step_through);
+
+	// Enter core scenarios, skipping intro:
+	add_debug_action({'@'}, "Quick-launch The Valley of Dying Things", []() -> void {debug_launch_scen("valleydy.boes");}, true);
+	add_debug_action({'#'}, "Quick-launch A Small Rebellion", []() -> void {debug_launch_scen("stealth.boes");}, true);
+	add_debug_action({'$'}, "Quick-launch The Za-Khazi Run", []() -> void {debug_launch_scen("zakhazi.boes");}, true);
```

Comment:
The `->void` part is unnecessary. Actually, I think `()` can be omitted too but I don't really care one way or another on that.